pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -170,27 +171,23 @@ class Search extends Component<IAppState, ISearchState> {
                         name={data[Search.KEY_TRANSIENTPATH]}
                         onChange={this.handleDeleteCheckboxChange}
                         key={'checked_' + key}
-                      />}
-
+                      />}                                   
                     <DataListItemCells key={"cells_" + key}
                       dataListCells={[
                         <DataListCell key={"title_" + key} width={2}>
                           {this.props.userAuthenticated &&
                             <Link to={data['pant:transientPath']} key={"link_" + key}>{data["jcr:title"]}</Link>}
                           {!this.props.userAuthenticated &&
                             <a href={"/" + data['pant:transientPath'] + ".preview"} target="_blank">{data["jcr:title"]}</a>}
+                        </DataListCell>,      
+                        <DataListCell key={"transient-source_" + key}>                          
```

Comment:
This isn't a BIG deal, but since I asked for other changes anyway, can you make sure that the `key` property is not only uniquely named, but also has something to do with its purpose? So in this case, change it from "transient-source_" to "published-date_" or something like that. Same thing for the 'date uploaded' field below. Please update attributes, especially `key` and `id` when appropriate as a general rule.

Reply:
Yes Ben, I will take care of this.
Even I feel these changes are necessary.